diagnostic.md

```diff
@@ -5,74 +5,83 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+This is a place to store the data and be able to support multiple users
+accessing the app.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+Model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
-```
+Controller```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+??
 ```
 
 What does C.R.U.D stand for?
 
 ```md
-// your response here
-```
+CREATE READ UPDATE DESTROY```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```md
-// your response here
+Controller
 ```
 
 List at least 5 standard rails actions that C.R.U.D requests correspond to?
 
 ```md
-// your response here
-```
+GET
+POST
+PATCH
+PUT
+DELETE```
```

Comment:
These are the HTTP verbs, we wanted the rails actions that they map to:

`Index, Show, Create, Update, Destroy`